pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -66,6 +66,17 @@ func isPort(ctx *domainContext, ifname string) bool {
 	return types.IsPort(ctx.deviceNetworkStatus, ifname)
 }
 
+//MountPoints - Holds mount target information which will be used when bringing up container
+type MountPoints struct {
+	targetPath string //Target path inside VM or container. Mandatory
+	fileSystem string //What type of file-system is expected to be mounted. Not mandatory, default: vfat
+	partition  int    //Which partition of the disk should be mounted. Not mandatory, default: 1
+}
+
+var FileSystems = map[string]int32{
```

Comment:
Why are you declaring a Global Variable? What is the value here? 

If needed, better use enums ( const ) instead of strings like "vfat"

Reply:
Removed FileSystems after discussion as there is no scope as of now to get this info from the user.